src/pages/ProductPage.jsx

```diff
@@ -0,0 +1,11 @@
+import SingleProduct from "../components/ProductPage";
+
+export default function ProductPage() {
+
+    return (
+      <div>
```

Comment:
How important is the div? 

It would be better if the Page fetched the data and passed it to `<SingleProduct product={product} />`